packages/edit-site/src/components/sidebar-navigation-screen-main/index.js

```diff
@@ -17,14 +17,19 @@ import SidebarNavigationScreen from '../sidebar-navigation-screen';
 import SidebarNavigationItem from '../sidebar-navigation-item';
 
 export default function SidebarNavigationScreenMain() {
-	const { navigationMenus } = useSelect( ( select ) => {
-		const { getEntityRecords } = select( coreStore );
-		return {
-			navigationMenus: getEntityRecords( 'postType', 'wp_navigation', {
-				per_page: -1,
+	const hasNavigationMenus = useSelect( ( select ) => {
+		const navigationMenus = select( coreStore ).getEntityRecords(
```

Comment:
Maybe add a comment to explain that it needs to be similar to the other request to avoid double network calls.